src/shared/model/Civic.ts

```diff
@@ -0,0 +1,27 @@
+export interface ICivicGeneData {
+    id: number;
+    name: string;
+    description: string;
+    url: string;
+    variants: {[variantName: string]: number};
+};
+
+export interface ICivicGeneVariant {
+    id: number;
+    name: string;
+    geneId: number;
+    description: string;
+    url: string;
+    evidence: {[evidenceType: string]: number};
+};
+
+export interface ICivicData {[name: string]: ICivicGeneData};
+
+export interface ICivicVariant {[id: string]: {[name: string]: ICivicGeneVariant}};
+
+export interface ICivicInstance {
```

Comment:
A class/interface name ending with `instance` feels a bit counter-intuitive.

Say an object is an instance of `ICivicInstance`, should we call that object a `ICivicInstance` instance in that case? :)